webhook_to_fedora_messaging_messages/github/github.py

```diff
@@ -41,12 +41,14 @@ def event_type(self):
     @property
     def summary(self):
         repo_name = self.body['body']['repository']['full_name']
-        return self.agent_name + " created " + self.event_name + (" on " + repo_name if repo_name is not None else "")
+        return self.agent_name + " created " + self.event_name + \
```

Comment:
It is preferrable to use f-strings, like `f"{self.agent_name} created {self.event_name}"`
You can do the test on a separate line:
```python
text = f"{self.agent_name} created {self.event_name}"
if repo_name is not None:
   text += f" on {repo_name}"
return text
```